vllm/v1/core/sched/scheduler.py

```diff
@@ -1015,11 +1015,7 @@ def _update_waiting_for_remote_kv(self, request: Request) -> bool:
         num_computed_tokens = min(num_computed_tokens, request.num_tokens)
         if num_computed_tokens == request.num_tokens:
             num_computed_tokens -= 1
-        self.kv_cache_manager.cache_blocks(
-            request,
-            self.kv_cache_manager.req_to_block_hashes[request.request_id],
-            num_computed_tokens,
-        )
+        self.kv_cache_manager.cache_blocks(request, num_computed_tokens)
```

Comment:
The `cache_blocks` method should only be called when `self.kv_cache_manager.enable_caching = True`.

Reply:
@hidva https://github.com/vllm-project/vllm/pull/19435 I included you as co-author.

Reply:
> To clarify, this comment isn't about the specific changes in this PR but rather an existing bug in the kv connector related changes here that you noticed that should be fixed?

Yes, I chose the PR corresponding to the most recent change on this line for convenience. Sorry about that.